src/components/Dialog/Dialog.scss

```diff
@@ -81,25 +81,30 @@
 
 .tm-w-xs,
 .tm-w-s {
-  .tm-dialog {
-    bottom: 10px;
-    left: 10px;
-    right: 10px;
-    width: calc(100% - 20px);
-    height: calc(40% - 10px);
-    max-height: 50%;
-    margin: auto;
-    transform: none !important; // block drag
+  .wkp-dialog {
+    .tm-rnd {
+      top: unset !important;
```

Comment:
forced to use '!important' to overwrite the new lib 'react-rnd' from react-ui new version.